terraform/azure_rke2_cluster/examples/gmsa.tfvars

```diff
@@ -75,19 +75,14 @@ apps = {
   }
 
   rancher-gmsa-plugin-installer = {
-    path      = "https://github.com/HarrisonWAffel/Rancher-Plugin-gMSA/raw/additional-fixes-refactors-and-docs/rancher-gmsa-plugin-installer-0.0.1.tgz"
+    path      = "https://github.com/HarrisonWAffel/Rancher-Plugin-gMSA/raw/main/assets/rancher-gmsa-plugin-installer/rancher-gmsa-plugin-installer-0.0.1.tgz"
```

Comment:
should be `rancher/` or `aiyengar2/`

Reply:
Changed to `aiyengar2` in an attempt to get this merged sooner rather than later